.readthedocs.yaml

```diff
@@ -0,0 +1,27 @@
+# This file is part of REANA.
+# Copyright (C) 2023 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+# Read the Docs configuration file
```

Comment:
Cosmetics: I would delete the extra comment with the documentation link (this can go into commit log message explaining _why_ we did the change); these links may be changing and are not necessary in the source code.  And I would also simplify the leading header comment:

```yaml
# ReadTheDocs version
version: 2
```

(Motivation: "Required" is indeed seen in the ReadTheDocs template https://docs.readthedocs.io/en/stable/config-file/v2.html but that is probably for education purposes... When we adapt the template, we may want to have a REANA-oriented look at the file structure.)